src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java

```diff
@@ -248,17 +249,18 @@ public CatalogObjectMetadata get(
         } else {
             List<CatalogObjectMetadata> metadataList;
             if (kind.isPresent() && contentType.isPresent()) {
-                metadataList = catalogObjectService.listCatalogObjectsByKindAndContentType(bucketName,
+                metadataList = catalogObjectService.listCatalogObjectsByKindAndContentType(Arrays.asList(bucketName),
```

Comment:
can we put this logic that are duplicated with CatalogObjectReportServoice.getListOfCatalogObjects()
in CatalogObjectService in some specific method?